components/StepBubbles/StepBubbles.tsx

```diff
@@ -10,9 +10,13 @@ type StepBubbleProps = {
    * Whether this bubble is the last in line and should not have a connector.
    */
   last?: boolean;
+  /**
+   * Whether the bubble should show a spinner if its active.
+   */
+  spinner?: boolean;
```

Comment:
Is the use case for the spinner to denote when a step is in progress? In that case, I might rename this prop `loading` and also add it up in the `StepBubbles` component too, then pass it thru to the `StepBubble`

e.g. `<StepBubbles steps={3} activeIndex={1} loading />` would show a spinner on the second step, with no need for a dev to manually set the `spinner` prop in `StepBubble` itself when consuming the exported `StepBubbles` component.

My rationale is that `StepBubble` isn't exported by this file, so I would think of it more like a private component whose implementation details are hidden. I would rename the prop to `loading` to more clearly indicate the intended use case and to abstract the idea of a loading state from its implementation (showing a `<Spinner />`).

Let me know your thoughts/if you have any questions!

Reply:
This makes a lot of sense yes. I updated the component to use loading from StepBubbles